### DISCUSSION TITLE
Not receiving verification email in my student email

### DISCUSSION THREAD

**Discussion by tysun** [community/community#23527] [New to GitHub]
<p>I am trying to register for the student pack but i am not recieving the github verifcation email.  Checked junk folder blah blah blah. Already sent a ticket to github but posting here as well incase it’s faster.</p>
<p>Please verifiy the <a href="http://aston.ac.uk" rel="nofollow noopener">aston.ac.uk</a> email address that’s in my github account.</p>
<p>Thanks </p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi tysun!</p>
<p>I’ve had a look and it appears that this is resolved. If you’re still having trouble, be sure to let us know.</p>

**Reply by simonevalerioti** [reply]
i have the same problem with
sim.valerioti@studuniroma3.onmicrosoft.com

**Comment by arionne00** [comment]
<p>I am having the exact same problem no verification email from my student email.</p>

**Reply by camerondodds-sv** [reply]
I'm having this same problem as well.

**Reply by camerondodds-sv** [reply]
I found the problem. My student email had message forwarding turned on, so my university blocked all emails to the account until I turned it off. They didn't want important documents forwarded to insecure emails. Hopefully this helps.

**Reply by VernSG** [reply]
> I found the problem. My student email had message forwarding turned on, so my university blocked all emails to the account until I turned it off. They didn't want important documents forwarded to insecure emails. Hopefully this helps.

so last solution is ask to college IT support?

**Comment by canuckjacq** [comment]
<p>:wave: Welcome <a class="mention" href="/arionne00">@arionne00</a></p>
<p>I’ve just triggered a new verification email. If you don’t get it, it’s worth double checking all your folders and inboxes just in case it got filed away as spam!</p>

**Comment by kir-** [comment]
<p>Hi i am having this problem too no verification email from my student email</p>

**Comment by queenofcorgis** [comment]
<p>Welcome <a class="mention" href="/kir">@kir</a> ! For student benefits and related conversations, we recommend visiting our <a href="https://github.com/orgs/github-community/discussions/9">Student Developer Pack FAQs post</a> as that may help answer your questions. If not, please use that Discussion to ask for assistance.</p>

**Comment by BaileyDead** [comment]
I'm having the same issue  as well

**Reply by BaileyDead** [reply]
also submitted a ticket but it's been 4 days and nothing

**Comment by Phoenixx028** [comment]
I have the same problem email is abhinav.5148@kvsrodelhi.in

**Comment by odion-cloud** [comment]
not receiving email

**Comment by antoinztte** [comment]
same here

**Comment by 0-chatGpt** [comment]
same here, for my other account

**Comment by CODE-Y02** [comment]
I am having same issue for account github.com/cndevil

**Comment by pistachioo00** [comment]
Same here, I've tried it 3x already but I still didn't receive any verification email from GitHub. I've checked my spam and refreshed it dozen of times but still no emails. Hope you can help

**Comment by AbdullahArpaci** [comment]
I am having a same issue

**Comment by NurKrasotka** [comment]
Hello. i have also problem with my veritification student email. Can u check mine please 20260819006@nevsehir.edu.tr
